model/action/AdminAction.php

```diff
@@ -117,4 +117,8 @@ public function getRequiredRights()
             'id' => 'GRANT'
         ];
     }
+    public function getWeight()
```

Comment:
https://oat-sa.atlassian.net/wiki/spaces/OAT/pages/872775905/Conventional+Commits
please use `feat: ` in the commit to create a proper version of extensions